HW#9/Functions.cpp

```diff
@@ -0,0 +1,113 @@
+
+#include <iostream>
+
+
+void  stringCharUppercaser(char inputChar[]) {
+
+	int i = 0;
+
+	while (inputChar[i] != '\0') {
+
+		if (isalpha(inputChar[i])) {
+			inputChar[i] = toupper(inputChar[i]);
+			i++;
+			continue;
+		}
+		else {
+			inputChar[i] = inputChar[i];
+			i++;
+		}
+
+	}
+
+	std::cout << "Result string: " << inputChar;
+
+}
+
+void  intTransformer(int inputNum[],int length) {
+
+	std::cout << "Result string: ";
+
+	for (int i = 0; i < length; i++) 
+	{
+		if (inputNum[i] > 0) {
+			inputNum[i] *= 2;
+			std::cout << inputNum[i] << " ";
+			continue;
+		}
+		else {
+			inputNum[i] = 0;
+			std::cout << inputNum[i] << " ";
+		}
+
+	}
+
+
+
+	
+
+}
+
+bool isPalindrome(char palindromeArray[])
+{
+
+	
+	for (int i = 0; i < strlen(palindromeArray); i++) {
```

Comment:
you can create variable to store strlen(palindromeArray). and use it here and inside of the loop